BREAKING_CHANGES.md

```diff
@@ -69,6 +71,30 @@ export const config: Config = {
 };
 ```
 
+#### Backwards-compat Configuration Fields Deprecated
+
+Several configuration options related to support for Safari <11, IE11, and Edge
+<19 have been marked as deprecated, and will be removed entirely in a future
+version of Stencil.
+
+##### dynamicImportShim
+
+The `extras.dynamicImportShim` option causes Stencil to include a polyfill for
+the `import()` function for use at runtime. The field is renamed to
+`__deprecated__dynamicImportShim` to indicate deprecation. To retain the
+prior behavior the new option can be set in your project's `stencil.config.ts`:
```

Comment:
We may want to clarify that this is for the dynamic `import()` fn, as opposed to the static variant - even though the name implies just that, I want to make sure we're as clear as possible/that folks don't think we dropped static imports (which would be _wild_ 😆)
```suggestion
The `extras.dynamicImportShim` option causes Stencil to include a polyfill for
the [dynamic `import()` function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) for use at runtime. The field is renamed to
`__deprecated__dynamicImportShim` to indicate deprecation. To retain the
prior behavior the new option can be set in your project's `stencil.config.ts`:
```

Reply:
Sure, that makes sense to me

definitely don't want people thinking we've decided to only support commonjs!

Reply:
haha no worries, just updated